component/background/StorageService.tsx

```diff
@@ -58,26 +59,31 @@ class StorageService {
    * @param key The key to remove
    * @return {Promise<{}>} A promise whose resolve takes no argument, which runs after the key is removed
    */
-  public remove(key): Promise<{}> {
+  public remove(key: string): Promise<{}> {
     return new Promise((resolve) => {
       this.storage.remove(key, resolve);
     });
   }
 
   /**
    * Subscribes a callback to an onChanged event.
-   * @param callback The function that takes one parameter of data.
+   * @param callback function that takes one parameter of data.
+   * @return  zero argument function that will unsubscribe the callback.
    */
-  public subscribe(callback): void {
-    chrome.storage.onChanged.addListener(callback);
-  }
+  public subscribe(callback) {
```

Comment:
I would be a good idea to specify the return type as
`public subscribe(callback) : () => void`